test/common/json/json_streamer_test.cc

```diff
@@ -134,9 +134,9 @@ TYPED_TEST(JsonStreamerTest, SubArray) {
 TYPED_TEST(JsonStreamerTest, TopArray) {
   {
     auto array = this->streamer_.makeRootArray();
-    array->addEntries({1.0, "two", 3.5, true, false, std::nan("")});
+    array->addEntries({1.0, "two", 3.5, true, false, std::nan(""), absl::monostate{}});
   }
-  EXPECT_EQ(R"EOF([1,"two",3.5,true,false,null])EOF", this->buffer_.toString());
+  EXPECT_EQ(R"EOF([1,"two",3.5,true,false,null,null])EOF", this->buffer_.toString());
```

Comment:
should we also have a unit test that directly calls addNull?

Reply:
not sure, because it will finally call the addNull of `Streamer`. And we also no direct test for other methods. 🤔

Reply:
add a test for all simple values.